### DISCUSSION TITLE
No dropdown on srcpf

### DISCUSSION THREAD

**Discussion by dylanrecinto** [codefori/vscode-ibmi#652] [IBM i] [answered]
Hi All,

Literally up until 15 minutes ago I was using Code for IBMi, stepped away from my desk and returned and am no longer able to access any of my CL/CLLE, RPG, PF sources. 

Anyone know what could have happened?

![image](https://user-images.githubusercontent.com/35200084/165318671-3fec22d6-fadb-4b33-b912-90db7aa57f7a.png)

Best, 
Dylan

**Comment by worksofliam** [comment, marked as answer]
Your filter should use `*SRCPF` for source code for now!

**Reply by dylanrecinto** [reply]
works! thank you!
